catalogue/webapp/pages/item.js

```diff
@@ -306,6 +333,10 @@ const ItemPage = ({
             src={urlTemplate({
               size: `max`,
             })}
+            alt={
+              (canvasOcr && canvasOcr.replace(/"/g, '')) ||
```

Comment:
What's the reason for this `replace`?

Reply:
if the text contains a double quote the browser will think that's the end of the attribute value

Reply:
Is that how it works?

As this is not template creation, but rather passing a value around, wouldn't it be the same as saying: 
```
const text = `things " that may or " may not have " in it`;
<img alt={text} />
```

Reply:
in the page source it comes out as <img alt="things " that may or " may not have " in it" />